terms/semisupervised_mm.tex

```diff
@@ -0,0 +1,99 @@
+\item Now let us consider also a large number of unlabelled instances\index{unlabelled instances} with unknown correct class labels\index{class labels}. We observe however, that they form two groups. Taking the assumption that instances in each class form a coherent group\index{coherent group}, we know that instances tend to center around a central mean\index{central mean} in a Gaussian distribution\index{Gaussian distribution}. 
+\item Through this assumption, the unlabelled data\index{unlabelled data} is capable of providing us with more information. We now notice that the two labelled instances\index{labelled instance} detailed in the first point are not the best examples for each of the classes in our dataset.
+\item Hence we take a semi-supervised\index{semi-supervised} estimate, which shows us that the decision boundary\index{decision boundary} should be between these two latter groups instead, at $x \approx 0.4$.
+\end{itemize}
+
+If the assumption holds, then using both the labelled\index{labelled} and unlabelled data\index{unlabelled data} gives us a more reliable decision boundary\index{decision boundary} than the one initially proposed. As we can see from this example (displayed also in Figure~\ref{fig:super_vs_semi}), the distribution of the unlabelled data\index{unlabelled data} helps us in identifying the regions having the same label, with the smaller amount of labelled instances\index{labelled instance} providing us with the actual labels.
+
+\begin{figure}
+\includegraphics[scale=1]{"graphics/semi-supervised learning/supervised_vs_semi"}
+\caption[Supervised vs Semi-Supervised Example][70pt]{An example of the decision boundaries determined by a Supervised system and a Semi-Supervised system.  Source: Zhu \& Goldberg (2009)}
+\label{fig:super_vs_semi}
+\end{figure}
+
+\subsection{Inductive vs. Transductive Semi-Supervised Learning}\label{sec:inductive_transductive}
+
+\newthought{In Semi-Supervised Learning}\index{semi-supervised learning}, on finds two modes. Since in our dataset\index{dataset} we have both labelled\index{labelled} and \index{unlabelled data}, we have two goals to achieve, namely:
+
+\begin{enumerate}
+\item To predict the labels for the upcoming data instances\index{data instance} in the test set\index{test set}. This is known as \textbf{Inductive Semi-Supervised Learning}\index{inductive semi-supervised learning}.
+\item To predict the labels of the unlabelled\index{unlabelled} data instances\index{data instances} inside of the training set\index{training set}. This is known as \textbf{Transductive Semi-Supervised Learning}\index{transductive semi-supervised learning}.
+\end{enumerate}
+
+Formally, in Inductive Semi-Supervised Learning \citep{zhu2009introduction}\index{inductive semi-supervised learning}, given a training sample; 
+
+\begin{equation}
+\{(\textbf{x}_i, y_i)\}^{l}_{i=1},\{\textbf{x}_j\}^{l+u}_{j=l+1}
```

Comment:
Absolutely NO !! `textbf` is used for text and not in equations.  Look at [this entry here](https://tex.stackexchange.com/questions/595/how-can-i-get-bold-math-symbols)